src/components/atoms/LanguageOption/index.tsx

```diff
@@ -1,16 +1,34 @@
-import * as React from 'react'
-import { StyleLanguageOptionContainer, StyleLanguageOption } from "./styles"
+import React, {useState} from 'react'
+import { StyleLanguageOptionContainer, StyleLanguageOption, StyleLanguageLink } from "./styles"
+import { americanFlagSvg, japanFlagSvg } from "../../../Svgs"
+import { useGetRedirectLanguage } from "../../../../localeRedirect"
 
 interface LanguageOptionInterface {
     language: string
 }
 
+const setLangVer = (props) => {
+    if (props.forcedLanguage === undefined || "ja") {
+        props.forcedLanguage = "en"
+        props.language = "en";
+        console.log("CHANGED PROPS: ", props.forcedLanguage)
+    } 
+    if (props.forcedLanguage === "en") {
+        props.forcedLanguage = "ja"
+        props.language = "ja";
+        console.log("CHANGED PROPS: ", props.forcedLanguage)
```

Comment:
Once you verify that it's working on production, don't forget to remove console log.